Auto-populate `POT-Creation-Date` header

vHeemstra opened the pull request:
Includes `POT-Creation-Date` by default.

Can be disabled by setting `options.noCreationDate` to `true`.

Fixes #45

rasmusbe commented:
Thank you for the PR! Please fix the style errors reported by `npm run test` and I'll merge and release a new version with the changes.

vHeemstra reviewed (commented):
Fixed styling

vHeemstra commented:
I refactored it a bit:
- corrected the styling issues
- renamed the option to be a bit more informative
- included the option in default options
- added option to the readme documentation

rasmusbe commented:
Awesome! Thanks!

marekdedic commented:
Correction: The option for this is actually called `includePOTCreationDate`.
